doc/UsersGuide/source/packagemanager.rst

```diff
@@ -1,70 +1,143 @@
-Package manager
-===============
-
-OpenModelica includes a package manager for installing Modelica packages.
-This chapter includes some details on how to install packages and how the package index itself works.
-
-Installing packages
--------------------
-
-The following commands setup an empty library directory for generating this documentation.
-You probably want to use the default ModelicaPath.
-
-.. omc-mos ::
-
-  clear()
-  system("rm -rf /tmp/omc-pkgman")
-  setEnvironmentVar("HOME","/tmp/omc-pkgman")
-  setModelicaPath("/tmp/omc-pkgman/.openmodelica/libraries/")
-
```

Comment:
This cannot be removed or we cannot use the package manager in the documentation. It could be commented after each command or put in a .. note :: or something

Reply:
Do you mean use the package manage here, or in some other places? For this part of the documentation, I wrote it to be self-contained, without the need of any part that actually executes to create it.

The reason I did that is that the stuff that you must write to get it to run on Jenkins is not exactly the same that you would write on your computer, and I find this really, really confusing. @sjoelund, it may not be for you, because you are used to that and you understand what it means and how to change it to run it locally, but we can't expect the average user, not to mention the newbie, to figure that out.